salt/metalk8s/repo/files/nginx.conf.j2

```diff
@@ -6,4 +6,7 @@ server {
         root   /var/www/repositories;
         autoindex on;
     }
+
+    include conf.d/{{ nginx_registry_config }};
+    include conf.d/{{ nginx_registry }};
```

Comment:
minor: `s/nginx_registry/nginx_registry_images` for clarity?

Reply:
I'm now including `*.inc*.